lib/index.js

```diff
@@ -28,12 +28,13 @@ function _defineProperty(obj, key, value) { if (key in obj) { Object.definePrope
 var Icon = function Icon(_ref) {
   var icon = _ref.icon,
       flex = _ref.flex;
-  return _react2.default.createElement('span', { className: (0, _classnames3.default)(_styles2.default.icon, _styles2.default[icon], _defineProperty({}, '' + _styles2.default.flex, flex)) });
+  return _react2.default.createElement('i', { className: (0, _classnames3.default)(_styles2.default.icon, _styles2.default[icon], _defineProperty({}, '' + _styles2.default.flex, flex)) });
```

Comment:
why is this an i tag when you haven't changed the source code to be an i tag?

Reply:
The build didn't change it... I'm launching it again.